crash when adding a stream mirror

```
nats stream add --mirror test
? Stream Name backup
? Storage backend file
? Retention Policy Limits
? Discard Policy Old
? Stream Messages Limit -1
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
main.(*streamCmd).prepareConfig(0xc0001b6b40, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/Users/matthiashanel/repos/natscli/nats/stream_command.go:1511 +0x1d6f
main.(*streamCmd).addAction(0xc0001b6b40, 0xc00013e6c0, 0x0, 0x0)
	/Users/matthiashanel/repos/natscli/nats/stream_command.go:1801 +0x65
gopkg.in/alecthomas/kingpin%2ev2.(*actionMixin).applyActions(0xc0003223d8, 0xc00013e6c0, 0x0, 0x0)
	/Users/matthiashanel/go/pkg/mod/gopkg.in/alecthomas/kingpin.v2@v2.2.6/actions.go:28 +0x6d
gopkg.in/alecthomas/kingpin%2ev2.(*Application).applyActions(0xc000176690, 0xc00013e6c0, 0x0, 0x0)
	/Users/matthiashanel/go/pkg/mod/gopkg.in/alecthomas/kingpin.v2@v2.2.6/app.go:557 +0xdf
gopkg.in/alecthomas/kingpin%2ev2.(*Application).execute(0xc000176690, 0xc00013e6c0, 0xc0003463c0, 0x2, 0x2, 0x0, 0x0, 0x0, 0xc000011e01)
	/Users/matthiashanel/go/pkg/mod/gopkg.in/alecthomas/kingpin.v2@v2.2.6/app.go:390 +0x95
gopkg.in/alecthomas/kingpin%2ev2.(*Application).Parse(0xc000176690, 0xc0000200b0, 0x4, 0x4, 0x1, 0xc000010a88, 0x0, 0x1)
	/Users/matthiashanel/go/pkg/mod/gopkg.in/alecthomas/kingpin.v2@v2.2.6/app.go:222 +0x228
main.main()
	/Users/matthiashanel/repos/natscli/nats/main.go:115 +0x1926
```

--- a/nats/stream_command.go
+++ b/nats/stream_command.go
@@ -1508,7 +1508,7 @@ func (c *streamCmd) prepareConfig() api.StreamConfig {
 		}
 	}
 
-	if c.maxMsgPerSubjectLimit == 0 && (len(c.subjects) > 0 || strings.Contains(c.subjects[0], "*") || strings.Contains(c.subjects[0], ">")) {
+	if c.maxMsgPerSubjectLimit == 0 && len(c.subjects) > 0 && (len(c.subjects) > 0 || strings.Contains(c.subjects[0], "*") || strings.Contains(c.subjects[0], ">")) {
 		c.maxMsgPerSubjectLimit, err = askOneInt("Per Subject Messages Limit", "-1", "Defines the amount of messages to keep in the store for this Stream per unique subject, when exceeded oldest messages are removed, -1 for unlimited. Settable using --max-msgs-per-subject")
 		kingpin.FatalIfError(err, "invalid input")
 		if c.maxMsgPerSubjectLimit <= 0 {

